sql/expression/function/registry.go

```diff
@@ -102,6 +102,7 @@ var BuiltIns = []sql.Function{
 	sql.Function2{Name: "gtid_subtract", Fn: NewGtidSubtract},
 	sql.Function1{Name: "hex", Fn: NewHex},
 	sql.Function1{Name: "hour", Fn: NewHour},
+	sql.Function0{Name: "icu_version", Fn: NewICUVersion},
```

Comment:
do we really need the intermediaries? can we just pre-fold the constant and return a literal here?